.gitignore

```diff
@@ -0,0 +1,14 @@
+# terraform metadata
+.terraform/
+.terraform.lock.hcl
```

Comment:
What is the reason behind placing the .terraform.lock.hcl in .gitignore? This is generally committed to the repo as suggested here: https://www.terraform.io/language/files/dependency-lock

Reply:
No reason, I just completely missed that 😓